command/bridge/finalize/finalize.go

```diff
@@ -77,13 +78,6 @@ func setFlags(cmd *cobra.Command) {
 		bridgeHelper.GenesisPathFlagDesc,
 	)
 
-	cmd.Flags().StringVar(
```

Comment:
Should we refactor the premine command to be similar to the finalize (remove the BladeManager addr and figure out on fly what is the address based on the chain id and bridge cfg)?